packages/styles/scss/patterns/blocks/content-block-mixedgroups/_content-block-mixedgroups.scss

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+@import '../../../globals/imports';
+
+@mixin content-block-mixedgroups {
+  .#{$prefix}--content-block-mixedgroups {
+    .#{$prefix}--content-group-cards {
+      @include carbon--breakpoint(lg) {
+        padding-left: $spacing-05;
+        padding-right: $spacing-05;
+      }
```

Comment:
What's going on here?

Reply:
`ContentGroup` has styles applied within it which work fine in an isolated Storybook story, but not so much when wrapped in another pattern. This needs to be fixed for several patterns.